subprojects/parseq-guava-interop/src/test/java/com/linkedin/parseq/guava/ListenableFutureUtilTest.java

```diff
@@ -0,0 +1,103 @@
+package com.linkedin.parseq.guava;
+
+import com.google.common.util.concurrent.ListenableFuture;
+import com.linkedin.parseq.Task;
+import java.util.concurrent.CancellationException;
+import java.util.concurrent.ExecutionException;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+/**
+ * Unit tests for {@link ListenableFutureUtil}
+ */
+public class ListenableFutureUtilTest {
+
+  @Test
+  public void testFromListenableFuture() {
```

Comment:
do you think we can easily add similar test case with an actual delayed setup? Something similar to what we do
https://github.com/linkedin/rest.li/blob/master/restli-client/src/test/java/com/linkedin/restli/client/TestParSeqBasedCompletionStage.java#L155

https://github.com/linkedin/rest.li/blob/master/restli-client/src/test/java/com/linkedin/restli/client/TestParSeqBasedCompletionStage.java#L181

Reply:
I don't think that is going to achieve anything. We will basically be testing that the behavior of listenablefuture and parseq task is consistent in a multi-threaded environment... which has already been tested in numerous places... so I am assuming it as an idiom :)